adoc/chapters/programming_interface.adoc

```diff
@@ -12525,6 +12525,182 @@ be tested using the [code]#has_known_identity# trait class.
 include::{header_dir}/identity.h[lines=4..-1]
 ----
 
+For each of the partial specializations listed in <<table.identities>>,
+[code]#known_identity# exists and has the value shown.
+
+[[table.identities]]
+.Known identities.
+[width="100%",options="header",separator="@",cols="25%,55%,20%"]
+|====
+@ Operator @ Available Only When @ Identity
+
+a@
+[source]
+----
+sycl::plus
+----
+a@
+[source]
+----
+std::is_arithmetic_v<AccumulatorT>
+----
+a@
+----
+AccumulatorT{}
+----
+
+a@
+[source]
+----
+sycl::multiplies
+----
+a@
+[source]
+----
+std::is_arithmetic_v<AccumulatorT>
+----
+a@
+----
+AccumulatorT{1}
+----
+
+a@
+[source]
+----
+sycl::bit_and
+----
+a@
+[source]
+----
+std::is_integral_v<AccumulatorT>
+----
+a@
+----
+~AccumulatorT{}
+----
+
+a@
+[source]
+----
+sycl::bit_or
+----
+a@
+[source]
+----
+std::is_integral_v<AccumulatorT>
+----
+a@
+----
+AccumulatorT{}
+----
+
+a@
+[source]
+----
+sycl::bit_xor
+----
+a@
+[source]
+----
+std::is_integral_v<AccumulatorT>
+----
+a@
+----
+AccumulatorT{}
+----
+
+a@
+[source]
+----
+sycl::logical_and
+----
+a@
+[source]
+----
+std::is_same_v<std::remove_v_t<AccumulatorT>, bool>
+----
+a@
+----
+true
+----
+
+a@
+[source]
+----
+sycl::logical_or
+----
+a@
+[source]
+----
+std::is_same_v<std::remove_v_t<AccumulatorT>, bool>
+----
+a@
+----
+false
+----
+
+a@
+[source]
+----
+sycl::minimum
+----
+a@
+[source]
+----
+std::is_integral_v<AccumulatorT>
+----
+a@
+----
+std::numeric_limits<AccumulatorT>::max()
+----
+
+a@
+[source]
+----
+sycl::minimum
+----
+a@
+[source]
+----
+std::is_floating_point_v<AccumulatorT> || std::is_same_v<std::remove_cv_t<AccumulatorT>, sycl::half>
+----
+a@
+----
+std::numeric_limits<AccumulatorT>::infinity()
```

Comment:
As pointed out by @fknorr in #163, the spec currently does not mandate that `std::numeric_limits` be specialized for `sycl::half`.

Reply:
> As pointed out by @fknorr in #163, the spec currently does not mandate that `std::numeric_limits` be specialized for `sycl::half`.

Perhaps we should?